pkg/cmd/generate-binary/main.go

```diff
@@ -216,6 +217,12 @@ var inetInputs = []string{
 	"192.168./10",
 }
 
+var arrayUUIDInputs = []string{
```

Comment:
Consider adding a test for `decimalArrayInputs` as well.

Reply:
Good idea, I just took the tests for decimal and arbitrarily grouped them into arrays.